### ISSUE TITLE
eslint: eslint whole folder fails on git bash

### ISSUE THREAD

**Issue by mikes-gh** [eslint/eslint#11662]
**Tell us about your environment**

* **ESLint Version:** 5.16.0
* **Node Version:** v10.15.3
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?**  
@typescript-eslint/parser

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports =  {
    parser:  '@typescript-eslint/parser',  // Specifies the ESLint parser
    extends:  [
      'plugin:@typescript-eslint/recommended',  // Uses the recommended rules from the @typescript-eslint/eslint-plugin
      'prettier/@typescript-eslint',  // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
      // 'plugin:prettier/recommended',  // Prettier rules which are better (eg long lines) but which prettier doesnt fix
    ],
    parserOptions:  {
      ecmaVersion:  5,  // Allows for the parsing of modern ECMAScript features
      sourceType:  'module',  // Allows for the use of imports
    },
    rules: { // TODO: The following rules are turned off but need to be addressed.
      '@typescript-eslint/no-use-before-define': 'none',
      '@typescript-eslint/explicit-function-return-type': 'none,',
      '@typescript-eslint/no-explicit-any': 'none',
      '@typescript-eslint/prefer-interface': 'none',
      '@typescript-eslint/array-type': 'none',
      '@typescript-eslint/no-angle-bracket-type-assertion': 'none',
      '@typescript-eslint/no-unused-vars': 'none',
      '@typescript-eslint/no-non-null-assertion': 'none',
      '@typescript-eslint/no-object-literal-type-assertion': 'none',
      //'@typescript-eslint/camelcase': 'none',
    },
  };
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

Lint whole project.

```js

```

run task from 
Terminal ->  Run Task.. menu item,
`eslint: eslint whole folder`

**What did you expect to happen?**

Work without error

**What actually happened? Please include the actual, raw output from ESLint.**
```
> Executing task: node_modules\.bin\eslint.cmd . <

/usr/bin/bash: node_modules.bineslint.cmd: command not found
The terminal process terminated with exit code: 127

Terminal will be reused by tasks, press any key to close it.
```
Running under PS or CMD works ok

**Are you willing to submit a pull request to fix this bug?**
Yes but haven't found issue yet.

**Comment by mikes-gh** [comment]
Sorry wrong repo
